source/reporting/index.html.md.erb

```diff
@@ -149,22 +149,15 @@ An example search request:
 
 Some of the query parameters you can use to search for payments are:
 
-* `reference`
-
-* `email`
-
-* `state`
-
-* `card_brand`
-
+* `reference` - case insensitive, exact match only
+* `email` - case insensitive
+* `card_brand` - case insensitive, exact match only
```

Comment:
Turns out the input for this doesn’t match the output in the response (e.g. you have to search for `master-card` to get `"card_brand": "Mastercard"`. But that disaster is probably not something to tackle now.

Reply:
Ah good to know - I've made a note to look at this later.